ts/Stock/Navigator/Navigator.ts

```diff
@@ -1503,8 +1503,10 @@ class Navigator {
                 chart.options.scrollbar,
                 { vertical: chart.inverted }
             );
-            if (!isNumber(options.margin) && navigator.navigatorEnabled) {
-                options.margin = chart.inverted ? -3 : 3;
+            if (navigator.navigatorEnabled) {
+                if (!isNumber(options.margin)) {
+                    options.margin = chart.inverted ? -3 : 3;
+                }
```

Comment:
Can we do it that way?

```suggestion
            if (navigator.navigatorEnabled && !isNumber(options.margin)) {
                  options.margin = chart.inverted ? -3 : 3;
```

Reply:
pewnie!